README.md

```diff
@@ -253,9 +253,9 @@ bootstrap({
 
 ### 3. Publish it to npm and use it!
 
-```
-npm install company-commit --save-dev
-./node_modules/.bin/company-commit
+```sh
+npm install company-commit --save-dev \
+  ./node_modules/.bin/company-commit
```

Comment:
I think that the trailing slash will make this get interpreted as: `npm install company-commit --save-dev ./node_modules/.bin/company-commit` by sh? It's intended to be two different commands...

Maybe move `--save-dev` and add a blank line to make it more obvious 🤔 

```suggestion
npm install --save-dev company-commit

./node_modules/.bin/company-commit
```

Reply:
Sorry about that! I read it wrong. I'll revert my changes when I get home tonight